core/src/services/iframe.js

```diff
@@ -91,6 +97,7 @@ export const navigateIframe = (config, component, node) => {
   } else {
     const goBackContext = component.get().goBackContext;
     config.iframe.style.display = 'block';
+    config.iframe.luigiNextViewUrl = viewUrl;
     config.iframe.contentWindow.postMessage(
```

Comment:
This postMessage should also have a specific target set. It should be the domain part of the iframe.luigiViewUrl.